tools/quake3/common/threads.c

```diff
@@ -551,7 +551,7 @@ void RunThreadsOn( int workcnt, qboolean showpacifier, void ( *func )( int ) ){
 		for ( i = 0 ; i < numthreads ; i++ )
 		{
 			/* Default pthread attributes: joinable & non-realtime scheduling */
-			if ( pthread_create( &work_threads[i], NULL, (void*)func, (void*)i ) != 0 ) {
+			if ( pthread_create( &work_threads[i], NULL, (void*)func, (void*)(size_t)i ) != 0 ) {
```

Comment:
`intptr_t` would be slightly more appropriate here: it's an integral type guaranteed to be able to store any object pointer.

Reply:
I'm doing it!